alpakka-zeromq/src/main/java/no/sysco/middleware/alpakka/zeromq/javadsl/internal/ZmqPublishStage.java

```diff
@@ -11,15 +11,15 @@
 import org.zeromq.ZMQ;
 import org.zeromq.ZMsg;
 
-public class ZeroMQPublishStage extends GraphStage<FlowShape<ZMsg, ZMsg>> {
+public class ZmqPublishStage extends GraphStage<FlowShape<ZMsg, ZMsg>> {
 
     private final String addresses;
 
     private final Inlet<ZMsg> inlet = Inlet.create("ZeroMQPublish.in");
```

Comment:
Nit: Name of the inlets/outlets should be renamed throught the code to match the renaming performed on class names (using Zmq instead of ZeroMQ)